packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -59,18 +73,36 @@ export class TableColumnNumberText extends TableColumnTextBase {
         this.updateColumnConfig();
     }
 
+    private decimalDigitsChanged(): void {
+        // decimalDigits is only used by the 'decimal' format option. Only update the
+        // column configuration if that is the current format option.
```

Comment:
It [doesn't look like the date-text column is this targeted](https://github.com/ni/nimble/blob/4574201ae49f8e30da7a0d87db78109f4236f340/packages/nimble-components/src/table-column/date-text/index.ts#L208) in updating its column config when settings change for a format other than the current one.

If you think this is worth doing, could we do it consistently for other column types? (I'm OK with this happening in a separate PR if that's cleaner). 

I'm also OK changing this to be unconditional like the date column is now.

Reply:
There isn't a good reason for that check here, so I'll remove it. I think that'll be better for consistency and to make the code paths more straightforward.